nixos/modules/services/web-apps/nexus.nix

```diff
@@ -6,9 +6,7 @@ let
 
   cfg = config.services.nexus;
 
-in
```

Comment:
Please make sure to format with `nixpkgs-fmt`.

Reply:
oh I did not know it was different from nixfmt

Reply:
There is currently no official formater for nixpkgs despite the name of nixpkgs-fmt. There is a formatting rfc in progress and the reference implementation can be found here:  https://github.com/serokell/nixfmt/pull/118